src/cljs/events.cljs

```diff
@@ -16,7 +16,7 @@
   (fn [{:keys [db]} _]
     {:db          (assoc db :loading true)
      :http-xhrio  {:method          :get
-                   :uri             "/api/test/1"
+                   :uri             "http://localhost:8081/api/test/1"
```

Comment:
Maybe add a global for the backend server `http://localhost:8081/api/`

Reply:
Done